test/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -121,6 +123,18 @@ internal void Create(TestAsset targetTestAsset, string testProjectsSourceFolder)
                 packageReferenceItemGroup = new XElement(ns + "ItemGroup");
                 projectXml.Root.Add(packageReferenceItemGroup);
             }
+            if (this.ReferencedProjects.Any())
```

Comment:
There's probably no need for the call to `Any()` if you're just going to loop through the list.  It's OK and simpler to just let it loop through a list with no elements.